src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -25,8 +25,7 @@ public class ConvertToJsonCommand : PSCmdlet
         [AllowNull]
```

Comment:
@iSazonov, your last commit had 36 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ConvertFrom-Json Unit Tests.Cannot round trip single element arrays without NoEnumerate switch
```powershell
Expected 1, but got '{}'.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertFrom-Json.Tests.ps1: line 128
128:         '[ 1 ]' | ConvertFrom-Json | ConvertTo-Json | Should -Be 1
```
ConvertFrom-Json Unit Tests.Can correctly round trip arrays with NoEnumerate switch set to $false
```powershell
Expected strings to be the same, but they were different.
Expected length: 5
Actual length:   7
Strings differ at index 1.
Expected: '[1,2]'
But was:  '[{},{}]'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertFrom-Json.Tests.ps1: line 116
116:         '[ 1, 2 ]' | ConvertFrom-Json -NoEnumerate:$NoEnumerate | ConvertTo-Json -Compress | Should -Be '[1,2]'
```
ConvertTo-Json.The result string is packed in an array symbols when AsArray parameter is used.
```powershell
Expected like wildcard '`[*1*]' to match '[
  {}
]', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertTo-Json.Tests.ps1: line 29
29:         $output | Should -BeLike "``[*1*]"
```
ConvertTo-Json.The result string is not packed in the array symbols when there is only one input object and AsArray parameter is not used.
```powershell
Expected strings to be the same, but they were different.
Expected length: 1
Actual length:   2
Strings differ at index 0.
Expected: '1'
But was:  '{}'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertTo-Json.Tests.ps1: line 37
37:         $output | Should -BeExactly '1'
```
ConvertTo-Json.Should handle null
```powershell
Expected strings to be the same, but they were different.
Expected length: 10
Actual length:   12
Strings differ at index 1.
Expected: '[1,null,2]'
But was:  '[{},null,{}]'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertTo-Json.Tests.ps1: line 73
73:         1, $null, 2 | ConvertTo-Json -Compress | Should -Be '[1,null,2]'
```